themes/gruvbox-dark-hard.json

```diff
@@ -1077,7 +1077,69 @@
     // OTHERS
     "textLink.foreground": "#83a598",
     "textLink.activeForeground": "#458588",
-    "debugToolBar.background": "#1d2021"
+    "debugToolBar.background": "#1d2021",
+    // GIT LENS
+    "gitlens.closedAutolinkedIssueIconColor": "#b16286",
+    "gitlens.closedPullRequestIconColor": "#cc241d",
+    "gitDecoration.addedResourceForeground": "#ebdbb2",
+    "gitlens.decorations.branchAheadForegroundColor": "#98971a",
+    "gitlens.decorations.branchBehindForegroundColor": "#d65d0e",
+    "gitlens.decorations.branchDivergedForegroundColor": "#d79921",
+    "gitlens.decorations.branchMissingUpstreamForegroundColor": "#cc241d",
+    "gitlens.decorations.statusMergingOrRebasingConflictForegroundColor": "#cc241d",
+    "gitlens.decorations.statusMergingOrRebasingForegroundColor": "#d79921",
+    "gitlens.decorations.workspaceCurrentForegroundColor": "#98971a",
+    "gitlens.decorations.workspaceRepoMissingForegroundColor": "#7c6f64",
+    "gitlens.decorations.workspaceRepoOpenForegroundColor": "#98971a",
+    "gitlens.decorations.worktreeHasUncommittedChangesForegroundColor": "#928374",
+    "gitlens.decorations.worktreeMissingForegroundColor": "#cc241d",
+    "gitlens.graphChangesColumnAddedColor": "#98971a",
+    "gitlens.graphChangesColumnDeletedColor": "#cc241d",
+    "gitlens.graphLane1Color": "#83a598",
+    "gitlens.graphLane2Color": "#458588",
+    "gitlens.graphLane3Color": "#d3869b",
+    "gitlens.graphLane4Color": "#b16286",
+    "gitlens.graphLane5Color": "#8ec07c",
+    "gitlens.graphLane6Color": "#689d6a",
+    "gitlens.graphLane7Color": "#fabd2f",
+    "gitlens.graphLane8Color": "#d79921",
+    "gitlens.graphLane9Color": "#b8bb26",
+    "gitlens.graphLane10Color": "#98971a",
+    "gitlens.graphMinimapMarkerHeadColor": "#98971a",
+    "gitlens.graphMinimapMarkerHighlightsColor": "#b8bb26",
+    "gitlens.graphMinimapMarkerLocalBranchesColor": "#83a598",
+    "gitlens.graphMinimapMarkerPullRequestsColor": "#fe8019",
+    "gitlens.graphMinimapMarkerRemoteBranchesColor": "#458588",
+    "gitlens.graphMinimapMarkerStashesColor": "#b16286",
+    "gitlens.graphMinimapMarkerTagsColor": "#7c6f64",
+    "gitlens.graphMinimapMarkerUpstreamColor": "#689d6a",
+    "gitlens.graphScrollMarkerHeadColor": "#b8bb26",
+    "gitlens.graphScrollMarkerHighlightsColor": "#d79921",
+    "gitlens.graphScrollMarkerLocalBranchesColor": "#83a598",
+    "gitlens.graphScrollMarkerPullRequestsColor": "#fe8019",
+    "gitlens.graphScrollMarkerRemoteBranchesColor": "#458588",
+    "gitlens.graphScrollMarkerStashesColor": "#b16286",
+    "gitlens.graphScrollMarkerTagsColor": "#7c6f64",
+    "gitlens.graphScrollMarkerUpstreamColor": "#8ec07c",
+    "gitlens.gutterBackgroundColor": "#fbf1c7",
+    "gitlens.gutterForegroundColor": "#282828",
```

Comment:
@azat-io could we try `#1d2021` here 🙏 ? it's the equivalent of `#282828` for hard contrast

I don't use gitlens so would trust your judgement after trying